### PR TITLE
06Dec rules update

### PR DESCRIPTION
**Pull request by lnazarenko**

### CODE REVIEW

File: training_data/de-DE/open_dis_words.csv [hunk 1]
```diff
@@ -89,7 +89,7 @@ Dorfmatratze,s,openly_discriminating,sexism,['-']
 Dorfnutte,s,openly_discriminating,sexism,['-']
 Dorfschlampe,s,openly_discriminating,sexism,['-']
 Dorftrampel,s,openly_discriminating,sexism,['-']
-Drache,s,openly_discriminating,sexism,['-']
+Drachen,s,openly_discriminating,sexism,['-']
```
Comment (lsmith77) [inline training_data/de-DE/open_dis_words.csv]: @tje-ww if you only want to trigger on the plural, you will need to make the word type `=s`

**Review by lsmith77 — APPROVED** [review]

File: training_data/de-DE/open_dis_words.csv (hunk 1 above)
Comment (tje-ww) [reply training_data/de-DE/open_dis_words.csv]: "Drachen" is a singular "Nebenform" of "Drache". I'd want to trigger on: Drache und Drachen as singular forms and their shared plural form. I've added a rule for the main-form spelling "Drache". How do I catch the subform singular "Drachen"?

Comment (lsmith77) [reply training_data/de-DE/open_dis_words.csv]: so `Drachen` is lemmatized to `Drache` .. so if you write the rule as `Drache,s,openly_discriminating,sexism,['-']` it works as intended.

Comment (lsmith77) [reply training_data/de-DE/open_dis_words.csv]: https://dev-54ta5gq-nfkxhzxe3xgbw.de-2.platformsh.site/debug/spacy?text=Drachen

Comment (tje-ww) [reply training_data/de-DE/open_dis_words.csv]: Am I correct to assume I can keep the "Drache" rule and delete the "Drachen" rule?

Comment (lsmith77) [reply training_data/de-DE/open_dis_words.csv]: exactly ..

**Pull request merged**